youtube_dl/extractor/seznamzpravy.py

```diff
@@ -0,0 +1,84 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import (
+    urljoin,
+    int_or_none,
+)
+
+
+class SeznamZpravyIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?seznam\.cz/zpravy/clanek/(?:[-a-z0-9]+)-(?P<id>[0-9]+)'
+    _API_URL = 'https://apiclanky.seznam.cz/'
+    _MAGIC_SUFFIX = 'spl2,2,VOD'
+
+    _TESTS = [{
+        # video with SDN URL
+        'url': 'https://www.seznam.cz/zpravy/clanek/jejich-svet-na-nas-utoci-je-lepsi-branit-se-na-jejich-pisecku-rika-reziser-a-major-v-zaloze-marhoul-35990',
+        'params': {'skip_download': True},
+        # ^ this is here instead of 'file_minsize': 1586, which does not work because
+        #   test_download.py forces expected_minsize to at least 10k when test is running
```

Comment:
I'd appreciate some hints on better work arounds minimal file size in tests. This video has an initializing segment that is 1586 bytes long, but I can't fill it here because [test_download.py:219](https://github.com/rg3/youtube-dl/blob/6d0630d8801fd3278a05fa7e55a73bd454403e5a/test/test_download.py#L219) seems to force the size to at least 10k.